### PR TITLE
Add accounting api support

### PR DESCRIPTION
**Pull request by kikettas**
### Description
Added support for Accounting API.

### How to test
In the example app, just pass the `.accounting` type in the `GiniVision.viewController(...api:)` method.

### Merging
Automatic

### CODE REVIEW

File: Example/Podfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ platform :ios, '9.0'
 use_frameworks!
 
 def project_pods
-    pod 'Gini-iOS-SDK'
+    pod 'Gini-iOS-SDK', :git => 'git@github.com:gini/gini-sdk-ios.git' ,:tag => '1.2.0-beta.1'
```
Comment (nsobadjiev) [inline Example/Podfile]: Is the git URL actually needed?

File: GiniVision/Classes/Networking/AccountingDocumentService.swift [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        self.giniSDK.sessionManager.logIn()
+    }
+    
+    func cancelAnalysis() {
+        if let document = document {
+            delete(document)
+        }
+        
+        analysisCancellationToken?.cancel()
+        resetToInitialState()
```
Comment (nsobadjiev) [inline GiniVision/Classes/Networking/AccountingDocumentService.swift]: Shouldn't these two lines be inside the if? Just for clarity?

File: Example/Podfile (hunk 1 above)
Comment (kikettas) [reply Example/Podfile]: I thought it wasn't but didn't work for me locally

File: GiniVision/Classes/Networking/AccountingDocumentService.swift (hunk 2 above)
Comment (kikettas) [reply GiniVision/Classes/Networking/AccountingDocumentService.swift]: No, the analysis can be cancelled when the document is being upload.

File: Example/Podfile (hunk 1 above)
Comment (nsobadjiev) [reply Example/Podfile]: Also, you should be using the HTTPS URL not the git one.

**Pull request merged**